pkg/client/tables.go

```diff
@@ -220,6 +220,7 @@ func (c *Client) NewTableEntry(
 		Action:          action,
 	}
 
+	//lint:ignore SA5011 if mfs==nil then for loop is not executed by default
```

Comment:
looks like your comment to disable the false-positive warning is not right as we still get the failure?

Reply:
I still don't understand why this line is wrong. Tried everything! The VSC lint implementation takes the line and removes the lint error but golangci-lint doesn't take it and gives error.... Archhh... :-(